FanartHandler/FanartSelectedOther.cs

```diff
@@ -304,6 +314,15 @@ public void RefreshSelected(RefreshWorker rw, System.ComponentModel.DoWorkEventA
               Utils.ContainsID(WindowsUsingFanartSelectedAwardMovie))
           {
             IsSelectedVideo = true;
+            if (Utils.iActiveWindow == 87266 || // Trakt Trending Movies
+                Utils.iActiveWindow == 87700 || // Trakt Calendar Movies
+                Utils.iActiveWindow == 87101 || // Trakt Popular Movies
+                Utils.iActiveWindow == 87263 || // Trakt Recommendations Movies
+                Utils.iActiveWindow == 87605 || // Trakt Anticipated Movies
+                Utils.iActiveWindow == 87607) // Trakt Box Office
+            {
+                IsSelectedVideo = false;
+            }

```

Comment:
I don’t quite understand why we reset the selected Video flag.

Reply:
I reset the IsSelectedVideo value because the clearart path is set with RefreshGenericSelectedProperties, then IsSelectedVideo calls EmptyAllProperties(false) to clear it.
The place this really happens in this flow of function calls EmptyAllProperties > EmptyVideoProperties > EmptySelectedMoviesProperties > ClearSelectedMoviePropertys.
The last function does this to clear one of the paths:
Utils.SetProperty("movie.clearart.selected", string.Empty);

I am open to ideas on how to better handle this.
Also, line 155 to 158 are only there to undo the impact caused by resetting the value. Left unchecked it makes the clearart appear on the prev and next pages.

Reply:
We could change this so that IsSelectedVideo is cleared for Non folders

Reply:
> We could change this so that IsSelectedVideo is cleared for Non folders. I would like to hear your ideas.

Folders can also contain `ClearArt`, `Disk`, `etc`. For example, `collections`, it is best not to clear anything, but not to return a value, then FH will clear them itself. I don’t remember from memory, but there is a simple mechanism: in one procedure we get all possible values, and then FH does everything for us.

Reply:
The issue here is with my current method of disabling IsSelectedVideo for clearart pages, when the item selected is not a movie or series, like page navigation it continutes to display the image because it calls AddSelectedMoviePropertys and still has the strIMDBID in memory.